common/starbase_buildings/hfe_starbase_building_population.txt

```diff
@@ -40,23 +38,27 @@ hfe_starbase_building_population_navigator = {
 				}
 			}
 		}
+		owner = { has_technology = hfe_tech_population_navigator }
```

Comment:
This trigger already in potential, why you need it again?

Reply:
I duplicated it to make sure. I'd rather not spend the time to remove it, but if that is what you really want, I will accommodate it.

Reply:
Okay, done